clustermesh/clustermesh.go

```diff
@@ -649,15 +649,19 @@ func (ai *accessInformation) validate() bool {
 		ai.ClusterID != "0"
 }
 
-func hasDeprecatedName(secretName string) bool {
-	if secretName == defaults.ClusterMeshServerSecretName ||
-		secretName == defaults.ClusterMeshAdminSecretName ||
-		secretName == defaults.ClusterMeshClientSecretName ||
-		secretName == defaults.ClusterMeshExternalWorkloadSecretName {
-		return true
+func getDeprecatedName(secretName string) string {
+	switch secretName {
+	case defaults.ClusterMeshServerSecretName:
+		return defaults.ClusterMeshServerSecretName + "s"
+	case defaults.ClusterMeshAdminSecretName:
+		return defaults.ClusterMeshAdminSecretName + "s"
+	case defaults.ClusterMeshClientSecretName:
+		return defaults.ClusterMeshClientSecretName + "s"
+	case defaults.ClusterMeshExternalWorkloadSecretName:
+		return defaults.ClusterMeshExternalWorkloadSecretName + "s"
```

Comment:
All of the above returns could be replaced by a single `return sercretName + "s"` in the end of the function.

Reply:
I've simplified this function as suggested.